lib/test/nightmare-service.js

```diff
@@ -51,6 +52,20 @@ module.exports = class NightmareService {
 
   prepareTestArgs(options) {
     const serveUrl = `http://${options.host || 'localhost'}:${options.port || '3000'}`;
+    // send report command first to produce coverage
+    this.nycArgs = ['report',
+      '--reporter=lcov',
+      '--reporter=text',
+      '--temp-directory',
+      './artifacts/coveragetemp',
+      '--clean'];
+    // use clean to delete past coverage runs
+    this.nycArgs.push();
+    this.testArgs = [];
+    const uiTestModule = require.resolve('@folio/ui-testing/test-module.js');
```

Comment:
Let's put these lines back into runNightmareTests() if possible:
```
const uiTestModule = require.resolve('@folio/ui-testing/test-module.js');
logger.log('test-module path', uiTestModule);
const mochaArgs = [uiTestModule].concat(this.testArgs);
```

While the values eventually make it into the same array, there is a separation of concerns here.  "prepareTestArgs()" is concerned with the individual args passed to "test-module" itself, whereas "runNightmareTests()" is concerned with how to invoke "test-module" via Mocha.  The latter's interest in file system paths really has nothing to do with how "test-module" operates internally and the arguments it expects.  Both are tested independently via the CLI's own unit tests.